app/src/main/res/layout/activity_match.xml

```diff
@@ -0,0 +1,76 @@
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:clipChildren="false">
+
+            <RelativeLayout
+                android:id="@+id/button_container"
+                android:orientation="horizontal"
+                android:layout_width="match_parent"
+                android:layout_marginBottom="8dp"
+                android:layout_height="80dp"
+                android:layout_alignParentBottom="true"
+                android:clipChildren="false">
+
+                <com.google.android.material.floatingactionbutton.FloatingActionButton
+                    android:id="@+id/skip_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_centerVertical="true"
+                    android:layout_toLeftOf="@+id/rewind_button"
+                    android:layout_toStartOf="@+id/rewind_button"
+                    app:fabSize="auto"
+                    app:rippleColor="#22ED7563"/>
+
+                <com.google.android.material.floatingactionbutton.FloatingActionButton
+                    android:id="@+id/rewind_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="32dp"
+                    android:layout_marginStart="32dp"
+                    android:layout_marginRight="32dp"
+                    android:layout_marginEnd="32dp"
+                    android:layout_centerInParent="true"
+                    app:fabSize="mini"
+                    app:rippleColor="#225BC9FA"/>
+
+                <com.google.android.material.floatingactionbutton.FloatingActionButton
+                    android:id="@+id/like_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_centerVertical="true"
+                    android:layout_toRightOf="@+id/rewind_button"
+                    android:layout_toEndOf="@+id/rewind_button"
+                    app:fabSize="auto"
+                    app:rippleColor="#226FE2B3"/>
+
+            </RelativeLayout>
+
+            <com.yuyakaido.android.cardstackview.CardStackView
+                android:id="@+id/card_stack_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_above="@+id/button_container"
+                android:padding="4dp"
+                android:clipToPadding="false">
+            </com.yuyakaido.android.cardstackview.CardStackView>
+
+        </RelativeLayout>
+
+    </LinearLayout>
+
+</androidx.drawerlayout.widget.DrawerLayout>
```

Comment:
Might be a good idea to add content description for the different FloatingActionButton